src/Features/LanguageServer/ProtocolUnitTests/References/FindAllReferencesHandlerTests.cs

```diff
@@ -292,6 +292,36 @@ public async Task TestFindAllReferencesAsync_StaticClassification()
             Assert.Equal(9, textRuns.Count());
         }
 
+        [Fact]
```

Comment:
These are the lsp tests. I would also expect a whole host of tests in the normal vb location that extensively tests the FAR engine.

Reply:
I failed to find a suitable location for hosting those tests, could you give a hint?

Reply:
I'm not at a computer, but search for FindReferencesTests. You'll find some vb ones with thousands of tests organized by type of symbol being searched for.

Reply:
Oh I failed to find them because they are in VB, didn't expect that. Will place my tests there then. Should I also migrate the one in the LSP over?

Reply:
i don't mind a single lsp test.